Components/Linux/wordpress-recipes-linux/install_wordpress.yml

```diff
@@ -0,0 +1,52 @@
+name: InstallWordPress
+description: this document installs Wordpress on Amazon Linux
```

Comment:
`This document installs WordPress on Amazon Linux 2` (nits for correct casing and the correct OS)